readthedocs/v3/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Do we need this?

Reply:
No. It's created by default when creating a Django app.